ui/desktop/src/components/settings/extensions/deeplink.ts

```diff
@@ -180,9 +180,9 @@ export async function addExtensionFromDeepLink(
     config.type === 'streamable_http' && config.headers && Object.keys(config.headers).length > 0;
 
   if (hasEnvVars || hasHeaders) {
-    console.log('Environment variables or headers required, redirecting to settings');
+    console.log('Environment variables or headers required, redirecting to extensions');
     console.log('Calling setView with:', { deepLinkConfig: config, showEnvVars: true });
-    setView('settings', { deepLinkConfig: config, showEnvVars: true });
+    setView('extensions', { deepLinkConfig: config, showEnvVars: true });
```

Comment:
we're now going to extensions whether you had this set or no. but in the second case we provide the extensionId. what's the difference? and also can we unify the two paths?